### PR TITLE
Bulk translation escaping

### PR DESCRIPTION
**Pull request by calellowitz**
@orangejenny 
This should work to avoid escaping quotes on the translation page without breaking the xml. I am not even sure we need to be escaping the < and > because i think all of it is re escaped on upload at the bottom of update_form_translation, but i left it there to be safe.

### CODE REVIEW

File: corehq/apps/app_manager/tests/test_bulk_app_translation.py [hunk 1]
```diff
@@ -297,7 +297,7 @@ class BulkAppTranslationDownloadTest(SimpleTestCase, TestXmlMixin):
         ('module1_form1',
          (('What_does_this_look_like-label', 'What does this look like?', '', 'jr://file/commcare/image/data/What_does_this_look_like.png', ''),
           ('no_media-label', 'No media', '', '', ''),
-          ('has_refs-label', 'Here is a ref <output value="/data/no_media"/> with some trailing text and bad &lt; xml.', '', '', '')))
+          ('has_refs-label', 'Here is a ref <output value="/data/no_media"/> with some trailing text and "bad" &lt; xml.', '', '', '')))
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_bulk_app_translation.py]: E501 line too long (136 > 115 characters)

**Comment by orangejenny** [comment]
Hey Cal, the unit tests pass, but forms aren't appearing correctly in vellum. This XML
```
<?xml version="1.0" encoding="UTF-8" ?>
<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
	<h:head>
		<h:title>Untitled Form</h:title>
		<model>
			<instance>
				<data xmlns:jrm="http://dev.commcarehq.org/jr/xforms" xmlns="http://openrosa.org/formdesigner/6A6958CC-027D-46C3-BBED-7D239EADD8E3" uiVersion="1" version="1" name="Untitled Form">
					<question1 />
					<question2 />
				</data>
			</instance>
			<bind nodeset="/data/question1" type="xsd:string" />
			<bind nodeset="/data/question2" type="xsd:string" />
			<itext>
				<translation lang="en" default="">
					<text id="question1-label">
						<value>uno &lt; dos's &amp; &gt; tres &amp; four and "five"</value>
					</text>
					<text id="question2-label">
						<value>before  हिन्दी there was <output value="/data/question1" /> later</value>
					</text>
				</translation>
			</itext>
		</model>
	</h:head>
	<h:body>
		<input ref="/data/question1">
			<label ref="jr:itext('question1-label')" />
		</input>
		<input ref="/data/question2">
			<label ref="jr:itext('question2-label')" />
		</input>
	</h:body>
</h:html>
```
Includes this question:
screen shot 2015-10-06 at 1 56 47 pm
With your branch checked out, I then go to bulk translations, download the file, reupload the file without changing it, and reopen the form in vellum to see this:
screen shot 2015-10-06 at 1 57 00 pm

**Comment by calellowitz** [comment]
@orangejenny i found that the same thing happens on master. is that the case for you as well?

**Comment by orangejenny** [comment]
Hrm. Indeed, master mucks up the greater-than sign, but it handles the ampersand and less-than sign correctly:
screen shot 2015-10-06 at 4 44 32 pm

**Comment by calellowitz** [comment]
@orangejenny I got the branch to behave like master by mimicking the escape function that was used before, but it still messes up that one greater than sign. im starting to think that it could be an error in the display code, because now the xml when viewing source matches the original

**Comment by orangejenny** [comment]
Looks fine. The issue with the greater-than sign is odd - it works fine in standalone vellum, but with vellum in HQ, even just using ">" as a label gets displayed as "&amp;gt;" on page refresh. Even though the form XML is identical in the two cases. @millerdev any ideas?

In any case, that's pre-existing. Merging this.

**Comment by millerdev** [comment]
It's odd that standalone vellum is different from vellum in HQ. It will likely take some digging to determine why. The code that does the conversion is somewhat magic as it uses the browser's HTML parser to convert between the display value and the escaped XML. I assume you did your tests on Chrome?

**Comment by orangejenny** [comment]
Yeah, my testing was all in Chrome.

**Pull request merged**